requirements.txt

```diff
@@ -26,11 +26,11 @@ pyasn1==0.1.8
 pycparser==2.14
 pymisp==2.4.54
 pymongo==3.0.3
-pyOpenSSL==0.15.1
+pyOpenSSL==16.2.0
 python-dateutil==2.4.2
 python-editor==0.3
 python-magic==0.4.6
-requests==2.7.0
+requests==2.13.0
```

Comment:
Doing `pip install -r`  with the current requirements from Debain testing fails, and can Break python environments due to incompatibility with the version of OpenSSL in use by the distribution.